README.md

```diff
@@ -105,6 +100,27 @@ called `add_footer`, you would:
 `git-hooks-core/whitelists` has been preconfigured with entries for the
 (initially empty) cred-alert whitelist in `whitelists.d`.
 
+## VERIFY IT ALL WORKS
+
+```
+$ cd ~/workspace/ANY-GIT-PROJECT   # or any workspace you are working on
+$ cp ~/.ssh/id_rsa* .
```

Comment:
Can you update this to use a fake credential instead? Attempting to commit real keys to validate it is working is more dangerous than it needs to be. The [install instructions](https://sites.google.com/a/pivotal.io/cloud-foundry/program-management/security/cred-alert-cli-instructions?pli=1) use a fake key.